.circleci/config.yml

```diff
@@ -169,10 +169,10 @@ jobs:
       - checkout
       - run:
           name: Install minimal doc and lint modules globally
-          command: npm i -g tslint lerna typedoc linkinator typescript gts tslint-consistent-codestyle tslint-microsoft-contrib
+          command: npm i -g tslint@5.0.0 lerna@3.17.0 typedoc linkinator typescript@3.7.2 gts@1.1.0 tslint-consistent-codestyle@1.16.0 tslint-microsoft-contrib@6.2.0
```

Comment:
Curious to know, why do we have to fix the version here?

Reply:
I try to understand why do we need to install some dependencies globally that are different then the one used in packages, then link them, instead of just do npm i. It seems like th gts was updated to version 2 few hours ago and then the dependencies that were used doesn't match. It fixed at first but very next commit was failing again. Currently trying to figure out it this can be fixed with certain version or I will remove those lines completely.

Reply:
This is done to avoid a slow lerna bootstrap for lint/docs which don't need most dependencies. We can go back to `lerna bootstrap --ci -- --only-dev` if it becomes a problem.